compiler/noirc_frontend/src/parser/parser/function.rs

```diff
@@ -280,4 +282,46 @@ mod test {
         assert_eq!(noir_function.parameters().len(), 1);
         assert!(noir_function.def.body.statements.is_empty());
     }
+
+    #[test]
+    fn parse_function_with_fv_attribute() {
+        let src = r#"#[requires(x > 2)]
```

Comment:
Tests generally should have three sections:

1. Setup
2. Invoking code that is being tested
3. Asserting result meets expectations

Please, restructure in this way.

Reply:
Done. I restructured all the tests except the last one. The last one is using the function `parse_all_failing()` which is defined by the upstream in the file `test_helpers.rs`. The function `parse_all_failing()` panics if a string from the vector is parsed correctly. This function is useful for testing multiple failing strings at once. I see no point in defining a new function which will do the same but will use `assert()` instead of `panic()`.

Reply:
Done.